src/main/kotlin/dev/haarolean/gnutardbot/abilities/ForwardSuggestionsAbility.kt

```diff
@@ -0,0 +1,43 @@
+package dev.haarolean.gnutardbot.abilities
+
+import dev.haarolean.gnutardbot.TardBot
+import dev.haarolean.gnutardbot.util.MemberUtils
+import org.telegram.abilitybots.api.objects.MessageContext
+import org.telegram.telegrambots.meta.api.methods.ForwardMessage
+import org.telegram.telegrambots.meta.api.methods.groupadministration.GetChat
+import org.telegram.telegrambots.meta.api.objects.Message
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException
+
+class ForwardSuggestionsAbility(private val bot: TardBot) : AbilityHandler {
+    private val memberUtils: MemberUtils = (MemberUtils(bot))
+
+    override fun isApplicable(ctx: MessageContext): Boolean {
+        val update = ctx.update()
+        val message: Message = when {
+            update.hasMessage() -> update.message
+            update.hasEditedMessage() -> update.editedMessage
+            else -> return false
+        }
+        if (bot.isAdmin(message.from.id)) return false
+        if (!message.isUserMessage) return false
+        val chat = bot.execute(GetChat(bot.properties.chatId))
+        if (!memberUtils.isUserKnown(chat.linkedChatId.toString(), message.from.id)) return false
+        return message.hasText() || message.hasPhoto() || message.hasVideo()
+    }
+
+    override fun handle(ctx: MessageContext) {
```

Comment:
Allowing users to send messages to the admins uncontrollably sounds like an easy target for spam. Before implementing this we have to implement the ability to ban the user from using the bot in this way if they abuse it. Also, please take a look at the built-in default abilities our bot library provides, there were a bunch of default commands including ones for banning people in personal bot chats.